pkg/monitoring/mimir/ingress/ingress.go

```diff
@@ -0,0 +1,36 @@
+package ingress
+
+import (
+	"os/exec"
+
+	"github.com/giantswarm/observability-operator/pkg/monitoring"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/giantswarm/observability-operator/pkg/monitoring/mimir"
```

Comment:
It's not used right?

Reply:
That's what vscode says but the file still needs it as it implements the function as a method from the `MimirService`

Reply:
Oh that's because you're not in the same package when you're defining a method but you're not really using any thing from the mimirservice so you can remove this `(ms *MimirService)` and make it a function instead

Reply:
Done